test/SlowTests/Issues/RavenDB-19448.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Server.Documents;
+using Sparrow;
+using Sparrow.Logging;
+using Tests.Infrastructure;
+using Tests.Infrastructure.Utils;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_19448 : ClusterTestBase
+    {
```

Comment:
Let's only check this manually without having a test.
Since logging tests tend to fail and they impact other tests during the execution.

Reply:
Removed